src/codegen.cpp

```diff
@@ -7011,13 +7017,16 @@ static std::pair<std::unique_ptr<Module>, jl_llvm_functions_t>
             info.line = jl_unbox_long(jl_fieldref(locinfo, 3));
             info.inlined_at = jl_unbox_long(jl_fieldref(locinfo, 4));
             assert(info.inlined_at <= i);
-            if (module == ctx.module)
-                info.is_user_code = mod_is_user_mod;
-            else
-                info.is_user_code = in_user_mod(module);
             info.file = jl_symbol_name(filesym);
             if (info.file.empty())
                 info.file = "<missing>";
+            if (module == ctx.module) {
+                info.is_user_code = mod_is_user_mod;
+                info.is_tracked = mod_is_tracked;
+            } else {
+                info.is_user_code = in_user_mod(module);
+                info.is_tracked = in_tracked_path(info.file);
+            }
```

Comment:
the module and file aren't necessarily related
```suggestion
            if (module == ctx.module)
                info.is_user_code = mod_is_user_mod;
            else
                info.is_user_code = in_user_mod(module);
            info.is_tracked = in_tracked_path(info.file);
```